hubconf.py

```diff
@@ -1,15 +1,15 @@
-from transformers import (
+from src.transformers import (
```

Comment:
This would actually be a good place for:

```python

import os
import sys

SRC_DIR = os.path.join(os.path.dirname(__file__), "src")
sys.path.append(SRC_DIR)
```

rather than making up a `src` package that creates an inconsistent import scheme from what we're using everywhere else, which may cause trouble in the future.